CIP-0027/README.md

```diff
@@ -53,6 +53,23 @@ A new tag of 777 is proposed for this implementation.  The community guidelines
 	}
 }
 
+## Example JSON with multiple addresses
+
+{
+	"777": [
```

Comment:
It's possible we may want the value in the "777" object to always be an array. However, the current implementation is an object so marketplaces would need to be backwards compatible anyway